src/components/social-share/github-icon/index.jsx

```diff
@@ -8,6 +8,7 @@ export const GitHubIcon = () => {
       href="https://github.com/JaeYeopHan/felog"
       className="github"
       aria-label="GitHub"
+      target="_blank"
```

Comment:
아 좋습니다 :)

```suggestion
      target="_blank"
      rel="noopener noreferrer"
```

요렇게 같이 추가해주면 어떨까요?